services/web/server/src/simcore_service_webserver/workspaces/_workspaces_db.py

```diff
@@ -132,9 +132,7 @@ async def list_workspaces_for_user(
             my_access_rights_subquery.c.my_access_rights,
         )
         .select_from(
-            workspaces.join(access_rights_subquery, isouter=True).join(
-                my_access_rights_subquery
-            )
+            workspaces.join(access_rights_subquery).join(my_access_rights_subquery)
```

Comment:
so at the end it was `isrouter=False`?

Reply:
no no this `outer` doesn't add any value so I just removed it.